clients/drcachesim/simulator/cache_simulator.h

```diff
@@ -63,8 +75,8 @@ class cache_simulator_t : public simulator_t {
     print_results() override;
 
     int_least64_t
-    get_cache_metric(unsigned level, metric_name_t metric, unsigned core = 0,
-                     cache_split_t split = DATA) const;
+    get_cache_metric(metric_name_t metric, unsigned level, unsigned core = 0,
```

Comment:
Please comment that a negative return value is an error and is of type stats_error_t